### PR TITLE
docs: add example of `useRuntimeConfig()` within a plugin

### PR DESCRIPTION
**Pull request by not-Ryan**
I'd like to propose an example for using runtime configuration within plugins.

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description
An example plugin that uses the runtime config. This is not well conveyed in the runtime plugins and so I thought this would be nice to have.

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/content/3.docs/1.usage/5.runtime-config.md [hunk 1]
```diff
@@ -77,6 +77,24 @@ const config = useRuntimeConfig()
 **`useRuntimeConfig` only works during `setup` or `Lifecycle Hooks`**.
 ::
 
+### Plugins
+
+If you want to use the runtime confg within any (custom) plugin, you can use `useRuntimeConfig()` inside of your `defineNuxtPlugin` function.
+
+For example:
+```ts
+import Urql from '@urql/vue';
```
[the remark this answers is not in the document]
Comment (not-Ryan) [reply docs/content/3.docs/1.usage/5.runtime-config.md:86]: Forgot to remove it. I shall do it directly.

File: docs/content/3.docs/1.usage/5.runtime-config.md [hunk 2]
```diff
@@ -77,6 +77,22 @@ const config = useRuntimeConfig()
...
+
+If you want to use the runtime confg within any (custom) plugin, you can use `useRuntimeConfig()` inside of your `defineNuxtPlugin` function.
+
+For example:
+```ts
+export default defineNuxtPlugin((nuxtApp) => {
+  const config = useRuntimeConfig();
+
+  const isServer = !!nuxtApp.ssrContext;
+  const url = isServer ? config.serverUrl : config.clientUrl;
```
Comment (danielroe) [inline docs/content/3.docs/1.usage/5.runtime-config.md:90]: Using `process.server` enables dead code removal in the build stage.

```suggestion
  const url = process.server ? config.serverUrl : config.clientUrl;
```

**Review by danielroe — APPROVED** [review]
Looks good to me - nice spot!

**Comment by pi0** [comment]
Thanks!

**Pull request merged**